Packs/Ironscales/Integrations/Ironscales/Ironscales.yml

```
@@ -0,0 +1,158 @@
+category: Data Enrichment & Threat Intelligence
+commonfields:
+  id: Ironscales
+  version: -1
+configuration:
+- defaultvalue: https://appapi.ironscales.com
+  display: Server URL (e.g. https://appapi.ironscales.com)
+  name: url
+  required: true
+  type: 0
+- defaultvalue: "10"
+  display: Maximum number of incidents per fetch
+  name: max_fetch
+  required: false
+  type: 0
+- display: API Key
+  name: apikey
+  required: true
+  type: 4
+- display: Company Id
+  name: company_id
+  required: true
+  type: 0
+- display: Scopes (e.g. "company.all")
+  name: scopes
+  required: true
+  type: 0
+- display: Trust any certificate (not secure)
+  name: insecure
+  required: false
+  type: 8
+- display: Use system proxy settings
+  name: proxy
+  required: false
+  type: 8
+- display: Fetch incidents
+  name: isFetch
+  required: false
+  type: 8
+- display: Incident type
+  name: incidentType
+  required: false
+  type: 13
+description: IRONSCLAES, a self-learning email security platform integration
+display: Ironscales
+name: Ironscales
+script:
+  commands:
+  - arguments:
+    - auto: PREDEFINED
+      description: Incident ID.
+      name: incident_id
+      predefined:
+      - ""
+      - ""
+      required: true
+    - description: Company id - Optional.
```

```suggestion
    - description: Company Id - If empty, the Company Id from the instance configuration is used.
```

Same for all other places.